src/couch_dist/src/couch_dist.erl

```diff
@@ -0,0 +1,108 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License. You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+-module(couch_dist).
+
+-export([childspecs/0, listen/2, accept/1, accept_connection/5,
+    setup/5, close/1, select/1, is_node_name/1]).
+
+
+childspecs() ->
+    {ok, [{ssl_dist_sup, {ssl_dist_sup, start_link, []},
+        permanent, infinity, supervisor, [ssl_dist_sup]}]}.
+
+
+listen(Name, Host) ->
+    NodeName = case is_atom(Name) of
+                   true -> atom_to_list(Name);
+                   false -> Name
+               end,
```

Comment:
Minor formatting nit: intend this block to be only 4 spaces in from the previous one.

```
NodeName = case is_atom(Name) of
     true -> atom_to_list(Name);
     false -> Name
end
```

Reply:
changed